src/components/Atoms/Text/Text.js

```diff
@@ -31,6 +31,11 @@ export const BaseText = styled.span`
           }
         `
     : null)};
+    ${({ mobileColor, theme }) => mobileColor && css`
+  @media ${theme.breakpoint('medium')} {
+    color: ${theme.color(mobileColor)};
```

Comment:
Be sure to check your work before pushing, dude; this now applies the mobile **_above_** this breakpoint, not below it.

Reply:
Much better!

In future, it might be best to think of our styles as mobile-first, so you'd apply any mobile stuff to the non-mediaquery part [here](https://github.com/comicrelief/component-library/blob/4de8fc339e28e01eb447b6a5d95b5da2627284f2/src/components/Atoms/Text/Text.js#L8) (handling if there is and isn't a supplied mobileColor prop), and then override it within a media query.

BUT this'll work just fine, good stuff :)